packages/devtools_app/lib/src/shared/feature_flags.dart

```diff
@@ -56,12 +53,6 @@ abstract class FeatureFlags {
   /// https://github.com/flutter/devtools/issues/4564.
   static bool widgetRebuildStats = true;
 
-  /// Flag to enable offline data on memory screen.
-  ///
-  /// https://github.com/flutter/devtools/issues/5606
-  static const bool memoryOffline =
-      _kMemoryOfflineExperiment; // requires special handling because it needs to be const
```

Comment:
can you leave the flag and just turn it to true so that it is easy to disable if we need to after rolling this out?